targets/post-common

```diff
@@ -35,7 +35,12 @@ if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
 fi
 
 # Add the primary user
-groups='audio,input,video,wayland,sudo,plugdev,crouton'
+if [ "${DISTROAKA:-"$DISTRO"}" = 'openSUSE' ]; then
+    groups='audio,input,video,wayland,root,plugdev,crouton'
```

Comment:
instead of two mostly-similar lists, why not put the common list above and just add sudo/root to groups in the if statements